jinja_utils_test.py

```diff
@@ -63,6 +63,17 @@ def test_parse_string(self):
         parsed_str = jinja_utils.parse_string('int {{i}}', {'i': 2})
         self.assertEqual(parsed_str, 'int 2')
 
+        # Invalid input string is used.
+        with self.assertRaises(Exception) as cm:
+            parsed_str = jinja_utils.parse_string('{{', {'a': 3, 'b': 0})
+        exception = cm.exception
+        expected_exception = 'Unable to parse string with Jinja: %s' % ('{{')
```

Comment:
You don't need to interpolate here, because the string is fixed. Just inline "{{" into the error message.

Also, use assertRaisesRegexp above in line 67, then you don't need this line and the line above.